website/content/docs/architecture/improving-consul-resilience.mdx

```diff
@@ -42,25 +37,21 @@ number of servers, quorum, and fault tolerance, refer to the
 [consensus protocol documentation](/consul/docs/architecture/consensus#deployment_table).
 
 Effectively mitigating your risk is more nuanced than just increasing the fault tolerance
-metric described above. You must consider:
-
-### Correlated Risks
-
-Are you protected against correlated risks? Infrastructure-level failures can cause multiple servers to fail at the same time. This means that a single infrastructure-level failure could cause a Consul outage, even if your server-level fault tolerance is 2.
-
-### Mitigation Costs
+metric described above. You must consider correlated risks at infrastructure-level - these are occasions which can cause multiple servers to fail at the same time. This means that a single infrastructure-level failure could cause a Consul outage, even if your server-level fault tolerance is 2.
 
 What are the costs of the mitigation? Different mitigation options present different trade-offs for operational complexity, computing cost, and Consul request performance.
 
-## Strategies to Increase Fault Tolerance
+## Fault tolerance
 
-The following sections explore several options for increasing Consul's fault tolerance.
+The following sections explore several options for increasing Consul's fault tolerance. For enhanced reliability, we recommend taking a holistic approach by layering these multiple functionalities together.
 
-HashiCorp recommends all production deployments consider:
-- [Spreading Consul servers across availability zones](#spread-servers-across-infrastructure-availability-zones)
-- <EnterpriseAlert inline /><a href="#use-backup-voting-servers-to-replace-lost-voters">Using backup voting servers to replace lost voters</a>
+- [Spread servers across infrastructure availability zones](#availability-zones)
+- <EnterpriseAlert inline /><a href="#quorum-size">Using backup voting servers to replace lost voters</a>

```

Comment:
The minimum quorum size does not imply backup servers. Those are two different features. 

```suggestion
- [Use a minimum quorum size to avoid performance impacts](#quorum-size).
```